hbase-server/src/main/java/org/apache/hadoop/hbase/master/region/MasterRegion.java

```diff
@@ -155,7 +158,16 @@ public RegionScanner getRegionScanner(Scan scan) throws IOException {
   }
 
   public FlushResult flush(boolean force) throws IOException {
-    return region.flush(force);
+    flusherAndCompactor.resetChangesAfterLastFlush();
```

Comment:
We add a line, but where do we reset this in tha past? We do not need to remove the original one?

Reply:
At present, the flush method is only called by test cases, reset is not called in this method, and there is no reset logic in the relevant test cases: TestMasterRegionWALCleaner/TestMasterRegionOnTwoFileSystems  and so on.

Can you help me make sure we don't need reset logic either?

Reply:
> We add a line, but where do we reset this in tha past? We do not need to remove the original one?

Do you mean the following changes are OK? 
`  public FlushResult flush(boolean force) throws IOException {
    flusherAndCompactor.resetChangesAfterLastFlush();
    return region.flush(force);
  }` 
  
  @Apache9  sir.

Reply:
I mean we have reset logic in MasterRegionFlusherAndCompactor, before calling this flush method. Now we move the reset logic into this method, then we should remove the reset logic in MasterRegionFlusherAndCompactor?

Reply:
Aha! I have understood what you mean.
I have a new commit again, could you take a look when you have time?
@Apache9  sir.

Reply:
We do not need to record the lastFlushTime here?